app/scripts/views/sms_sent.js

```diff
@@ -8,39 +8,64 @@
 define(function (require, exports, module) {
   'use strict';
 
+  const _ = require('underscore');
+  const BackMixin = require('views/mixins/back-mixin');
   const BaseView = require('views/base');
   const Cocktail = require('cocktail');
   const CountryTelephoneInfo = require('lib/country-telephone-info');
+  const { FIREFOX_MOBILE_INSTALL } = require('lib/sms-message-ids');
   const { MARKETING_ID_AUTUMN_2016 } = require('lib/constants');
-  const MarketingMixin = require('views/mixins/marketing-mixin');
+  const MarketingMixin = require('views/mixins/marketing-mixin')({ marketingId: MARKETING_ID_AUTUMN_2016 });
+  const ResendMixin = require('views/mixins/resend-mixin')({ successMessage: false });
   const Template = require('stache!templates/sms_sent');
 
-  function formatPhoneNumberForCountry (phoneNumber, country) {
-    const countryInfo = CountryTelephoneInfo[country];
-    return countryInfo.format(phoneNumber);
-  }
+  const t = msg => msg;
+
+  const UNTRANSLATED_RESENT_MESSAGE = t('Download link resent to %(phoneNumber)s');
```

Comment:
@ryanfeeley asked me to change the other text from `Download link` to `App link`, we should do the same here.